src/OnboardingSPA/steps/SiteGen/SiteLogo/index.js

```diff
@@ -81,6 +95,9 @@ const SiteGenSiteLogo = () => {
 			site_logo: siteLogoNew.id,
 		} );
 		setSiteLogo( siteLogoNew );
+		trackOnboardingEvent( new OnboardingEvent( ACTION_LOGO_ADDED ), {
+			source: SITEGEN_FLOW,
+		} );
```

Comment:
Can we determine if it was added or deleted?

Reply:
Reporting on clicking `Next`.